fluss-filesystems/fluss-fs-gs/src/test/resources/fake-service-account.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "service_account",
+  "project_id": "test-project",
+  "private_key_id": "a6746181-3dfc-4ad2-9341-d3b42ae2b57f",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDlZ3Ar/SqoQ8Ou\nV5J5fi8LKXNxRd72w1YvkbUM3QiKqUmQI9E89ILxXFpiChb02/1p2M03ZsrWFNS3\n9b9PiVbj50m9AWRTzPoGaYgR4+dXBBX+fI8r6+NrY4rrFhuaANidFO4SWf02Rmpo\nqM7pwDuHtYXE4uFt+50bl9PaoVSbHYDLyhvGe5lyMOTgA7sEShhh4GWror/hdIzX\nIukEbeW6wFaPMwdT8VdPl1NV7F8IEpHRL8fkw7f52B05xhEULci3kG99a3trqZc3\n1vgjHjNolFuvrlgp7C3l1VTpoB2gnZLnviYadXAPmntX/6gyZjbMuwWxWVsHoESM\nQFiV/7oVAgMBAAECggEAHaowWnPgwAURSRp4bb8IQNKBo+hz4meUnc6jAbCUge/N\nvDaxgEcjJqh5ZJum58+J19XqBavEd4YWktpTvKqHIq3+PlNvgJmtTEZ5ysx0li9I\nU9ISJo0YqcKjzG4dsB7DZB64gYWDBbF54aJQrXAHEq/BxlnLanBC2vZZdjhuCye5\nCDPUCN80KXQhbJbAn6XLkKnzTWeT5TBvzvTbnx9v+dnwsv7KCwWRAbK+oQg/krWL\nsZA7cKu3WJQNAAq7MCalrTRvI8Yxi9xwAglTy4tcJDdEz6feKoWizXmKgfq/QRFZ\nAgacjQxKUVnODri2vB96brdZo3bZcL5mDlAqtVIjuQKBgQD+TW5rc0pWdBDG/fXn\ngX83ZvmhdlIYh+7vemA5JVy7PnGcjhn+W0RAwX/E1ybnFHfgJ9Ca4kEqtDujVyJU\nD4VWsSC+3ocL/t49kLnbfrhiNr2TsSUWL/+BKx8E1KaljeyKQjbw420cvOEpGtZc\nzrd9MiYb84C8HbRxMLGrfwiJGQKBgQDm73V0v/4BnJHGk7p+I+vOVff6ZapkCiJr\n2R6HufC+LvLusBrHQs0+NZeHIZ+Ak7riBTRXdi81jGAJTLLCc5eOoOIew/31VxdA\nDYWPGhDeT1jFM88cMK5uHwctsqBs/CVtlp/wFo6MaSv9wXBUFiQrbUrTh5gV58YW\naWDGfpbMXQKBgQCSQnyYhjWiJZ0kphoRbXR6q9o1WDT9cOkmiS7gjzGXa+GyFjSz\nC1DlIaVX1BrG2wnbPWsIOcsbQY/vcQUIbejVpIhd3uSzLIIrzJpouKhZrHw6fq8C\n5NZCN0fe4mN6YndCSbUJtPQjjCrAFnRl6mq0MeyjNNi82STX2RbGuQhyuQKBgCax\ntseCJ+hTOKXCOPxLvoYLJ4rAICW1AQF7Fr99K1ZifXf67/G1QriKuEH/Iz+jASw/\n3P5OoihhxfGo9UbZoWpXkKvBATtrhUzJNzi8EFl/iAs63BgIHCH9dvF0PaclNIxB\nMblDvkzOjn1tPQoJz7OrE1+QF66L8EfvDjUUHuZNAoGBANlCXeooDE/VRSSK9Ems\nKnshPiyRvALnVDtaKM7pMTGXrZVCTgdBW/M4oGlpeUz8Fv59QqHld2IJG8XNuR+A\nr/Acq9/5GboBN7RATOcZILhje2IaAd4dtsVUU1qWe759lMYMi8f3eSl2cEtDYAmy\nXWq/l8ZIM26met5ubBGMqA/2\n-----END PRIVATE KEY-----",
```

Comment:
Is it a real `private_key`. Won't it be dangerous to expose it directly in code?

Reply:
This is indeed a private key but not a private key generated by a cloud provider with any permissions attached to it.

I generated this key locally using 
```
openssl genrsa -out key.pem 2048
openssl rsa -in key.pem -outform PEM -pubout -out public.pem
```
Thus it is a dummy private key, for the purpose of testing.